src/Symfony/Component/Process/Process.php

```diff
@@ -1108,11 +1108,13 @@ public function setStdin($stdin)
      *
      * This content will be passed to the underlying process standard input.
      *
-     * @param string|null $input The content
+     * @param mixed $input The content
      *
      * @return self The current Process instance
      *
      * @throws LogicException In case the process is running
+     *
+     * @deprecated Passing an object as an input is deprecated since version 2.5 and will be removed in 3.0.
```

Comment:
this does not look fixed. IDEs will still warn about using a deprecated method (there is no phpdoc to deprecate only a specific signature AFAIK)